.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
```

Comment:
Please remove this file from the PR and only keep your .json model file.

Reply:
Has been deleted.